lambdatimes/src/components/Content/Cards.js

```diff
@@ -1,15 +1,21 @@
-import React, { Component } from 'react';
-import Card from './Card';
+import React, { Component } from "react";
+import Card from "./Card";
+import PropTypes from "prop-types";
 
 const Cards = props => {
   return (
     <div className="cards-container">
+      {props.cards.map(i => (
```

Comment:
This is good, but it would make more sense, and is best practice to use a descriptive variable name, `i` means nothing, and actually would be more indicative of `index`, but it _isn't_ an index, it's the actual element of the current iteration. Something like calling it `card` here, would be perfect.